File: Sumofdigit.py
#  You're given an integer N. Write a program to calculate the sum of all the digits of N.
n = int(input("How many times you want to perform this test ?\n "))
for _ in range(n):
    num = input("Enter a number: ")
    sum = 0
    for i in num:
        sum += int(i)
    print(sum)

File: sumOfdigit.cpp

// You're given an integer N. Write a program to calculate the sum of all the digits of N.

#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cout<<"Enter number of test cases: ";
    cin>>n;
    while(n--)
    {
     int num,m,sum=0;
     cout<<"Enter a number: ";
     cin>>num;
     while(num>0)
     {
      m=num%10;
      sum+=m;
      num/=10;
     }
     cout<<sum<<"\n";
    }
    return 0;
}
